Source/ApiTemplate/build.cake

```diff
@@ -1,3 +1,5 @@
+#addin "nuget:?package=Cake.MinVer&version=1.0.0-rc0001"
```

Comment:
This needs to be:

```c#
//#if (Docker)
 #addin "nuget:?package=Cake.MinVer&version=1.0.0-rc0001" 

//#endif
```

Reply:
Does that apply to all 3 scripts? I'll wait for you to decide if you'll move forward with it